testutils/mocks/Logger.go

```diff
@@ -0,0 +1,81 @@
+package mocks
```

Comment:
I was paving the way to move `test_utils.go` into a separate package inside `testutils/`, with the view that these test utilities should be used by all sub-packages.